.asf.yaml

```diff
@@ -45,3 +45,19 @@ github:
     squash_commit_message: PR_TITLE_AND_DESC
     merge: false
     rebase: false
+
+  protected_branches:
+    trunk:
+      required_status_checks:
+        strict: false
+        contexts:
+          - build / CI checks completed
+      required_pull_request_reviews:
+        required_approving_review_count: 1
+
+    # Disable force push on release branches
```

Comment:
So we will not require this check on non-trunk release branches?

Reply:
Correct. Otherwise, we would need to backport a lot of build changes.